.circleci/config.yml

```diff
@@ -171,6 +179,10 @@ jobs:
           name: 'Save core dumps'
           path: /tmp/core_dumps
           when: on_fail
+      - store_artifacts:
+          name: 'Save pg_upgrade logs for newData dir'
+          path: /tmp/pg_upgrade_newData_logs
+          when: on_fail
```

Comment:
I think this line doesn't do anything in store_artifacts step. So you can remove it (I know it exists with other store_artifacts but we should also remove them separately

Reply:
you meant `when: on_fail`, right ?

Reply:
Yes